vercel/next.js
# bug fix on a pull request #64911

### What?

Fixes a bug of removed dependencies by accident, which caused the example not to install dependencies. 

### How to test?

Running this script should initiate a project that works correctly:
```
 pnpm create next-app --example with-storybook with-storybook-app
```


Fixes this pull request https://github.com/vercel/next.js/pull/64911
